frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx

```diff
@@ -49,13 +55,23 @@ export const TicketsList = ({ tickets }: TicketsListProps) => {
 		setSelectedTemplates(new Set(selectedTemplates));
 	};
 
-	const getTicketsByTemplateId = (templateId: string) => tickets.filter(({ type }) => type === templateId);
+	const filterCompleted = (ticket) => {
+		const issuePropertyStatus = get(ticket, 'properties.Status');
+		const treatmentStatus = get(ticket, 'modules.safetibase.Treatment Status');
+
+		const isCompletedIssueProperty = [TicketStatuses.CLOSED, TicketStatuses.VOID].includes(issuePropertyStatus);
+		const isCompletedTreatmentStatus = [TreatmentStatuses.AGREED_FULLY, TreatmentStatuses.VOID].includes(treatmentStatus);
 
-	const getFilteredTickets = () => {
-		if (selectedTemplates.size === 0) return tickets;
-		return flatMap([...selectedTemplates], getTicketsByTemplateId);
+		return (isCompletedIssueProperty || isCompletedTreatmentStatus) === showingCompleted;
 	};
 
+	const getTicketsByTemplateId = (templateId: string) => tickets.filter(({ type }) => type === templateId);
+
+	const getFilteredTickets = (() => {
+		if (selectedTemplates.size === 0) return tickets.filter(filterCompleted);
+		return flatMap([...selectedTemplates], getTicketsByTemplateId).filter(filterCompleted);
+	})();
+
 	const getTemplatesForFilter = () => templates.filter(({ _id }) => getTicketsByTemplateId(_id).length > 0);
```

Comment:
I think this should be changed to a state variable.
Also availableTemplates has all the information to be rendered

```suggestion
useEffect( () => {
availableTemplates = templates.reduce((partial, { _id, name }) => {
		const { length } = tickets.filter(({ type }) => _id === type);
		if (!length) return partial;
		return [...partial, { _id, name, length }];
	}, []);
	}, [tickets, templates]);


```